validation/stokes_drift/surface_wave_quasi_geostrophic_flow.jl

```diff
@@ -98,18 +108,56 @@ n = Observable(1)
 
 un = @lift interior(ut[$n], :, :, Nz)
 wn = @lift interior(wt[$n], :, :, Nz)
+en = @lift interior(et[$n], :, :, Nz)
 
 xu, yu, zu = nodes(ut)
 xw, yw, zw = nodes(wt)
+xe, ye, ze = nodes(et)
+
+Nx, Ny, Nz = size(grid)
+xa, ya, za = nodes(grid, Center(), Center(), Center())
+xc, yc, zc = nodes(grid, Center(), Center(), Center())
+xa = reshape(xa, Nx, 1)
+ya = reshape(ya, 1, Ny)
+
+An = @lift begin
+    t = times[$n]
+    ξ = @. xa - cᵍ * t
+    A.(ξ, ya)
+end
+
+xu = xu .* 1e-3
+yu = yu .* 1e-3
+zu = zu .* 1e-3
+xw = xw .* 1e-3
+yw = yw .* 1e-3
+zw = zw .* 1e-3
+xe = xe .* 1e-3
+ye = ye .* 1e-3
+ze = ze .* 1e-3
 
-fig = Figure(size=(800, 400))
+xc = xc .* 1e-3
+yc = yc .* 1e-3
+zc = zc .* 1e-3
 
-axu = Axis(fig[1, 1], xlabel="x (m)", ylabel="z (m)", aspect=1)
-axw = Axis(fig[1, 2], xlabel="x (m)", ylabel="z (m)", aspect=1)
+fig = Figure(size=(1600, 400))
 
-heatmap!(axu, xu, yu, un)
-heatmap!(axw, xw, yw, wn)
+axu = Axis(fig[1, 1], xlabel="x (km)", ylabel="z (km)", aspect=2, title="u")
+axw = Axis(fig[1, 2], xlabel="x (km)", ylabel="z (km)", aspect=2, title="w")
+
+ulim = 1e-2 * Uˢ
+elim = ulim^2
+wlim = 1e-1 * Uˢ / (δ * 2k)
+
+heatmap!(axu, xu, yu, un, colormap=:balance, colorrange=(-ulim, ulim))
+#heatmap!(axu, xe, ye, en, colormap=:solar, colorrange=(0, elim))
+contour!(axu, xc, yc, An, color=:gray, levels=5)
+
+heatmap!(axw, xw, yw, wn, colormap=:balance, colorrange=(-wlim, wlim))
+contour!(axw, xc, yc, An, color=:gray, levels=5)
 
 record(fig, "surface_wave_quasi_geostrophic_induced_flow.mp4", 1:Nt, framerate=8) do nn
+#record(fig, "surface_wave_non_rotating_induced_flow.mp4", 1:Nt, framerate=8) do nn
```

Comment:
why comment the movie?
```suggestion
#record(fig, "surface_wave_non_rotating_induced_flow.mp4", 1:Nt, framerate=8) do nn
```

Reply:
```suggestion
record(fig, "surface_wave_non_rotating_induced_flow.mp4", 1:Nt, framerate=8) do nn
```